packages/blocks/src/page-block/edgeless/components/component-toolbar/component-toolbar.ts

```diff
@@ -254,8 +265,17 @@ export class EdgelessComponentToolbar extends WithDisposable(LitElement) {
     return html`<component-toolbar-menu-divider></component-toolbar-menu-divider>`;
   }
 
-  private async _updatePosition() {
-    // await this.updateComplete;
+  protected override updated(_changedProperties: PropertyValues): void {
+    // if (_changedProperties.has('left') || _changedProperties.has('top')) {
+    const [left, top] = this._updatePosition();
```

Comment:
Generally an `updateSomething` API indicates performing side effect, instead of returning values. Using naming like `computePosition` could be better. Also note not having the redundant comment.

Reply:
ok, I forgot to change the name